clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/GvcfTests.scala

```diff
@@ -47,7 +47,8 @@ trait GvcfTests { self: BaseIntegrationSpec =>
       "--version",
       key.version.toString,
       "--metadata-location",
-      tmpMetadata.toString
+      tmpMetadata.toString,
+      "--force-update"
```

Comment:
Can we parameterize the `runUpsertXXX` methods, so we can write int tests for upsert-ing without the force flag?